src/game/logic/object/weapon.cpp

```diff
@@ -457,3 +461,495 @@ bool Weapon::Is_Source_Object_With_Goal_Position_Within_Attack_Range(
     return false;
 #endif
 }
+
+const char *s_theDeathTypeNames[] = {
+    "NORMAL",
+    "NONE",
+    "CRUSHED",
+    "BURNED",
+    "EXPLODED",
+    "POISONED",
+    "TOPPLED",
+    "FLOODED",
+    "SUICIDED",
+    "LASERED",
+    "DETONATED",
+    "SPLATTED,",
+    "POISONED_BETA",
+    "EXTRA_2",
+    "EXTRA_3",
+    "EXTRA_4",
+    "EXTRA_5",
+    "EXTRA_6",
+    "EXTRA_7",
+    "EXTRA_8",
+    "POISONED_GAMMA",
+    nullptr,
+};
+
+const char *s_theAutoReloadsClipNames[] = {
+    "YES",
+    "NO",
+    "RETURN_TO_BASE",
+    nullptr,
+};
+
+const char *s_theAttackTypeNames[] = {
+    "PER_SHOT",
+    "PER_ATTACK",
+    "PER_CLIP",
+    nullptr,
+};
+
+const char *s_theRadiusDamageAffectsNames[] = {
+    "SELF",
+    "ALLIES",
+    "ENEMIES",
+    "NEUTRALS",
+    "SUICIDE",
+    "NOT_SIMILAR",
+    "NOT_AIRBORNE",
+    nullptr,
+};
+
+const char *s_theProjectileCollidesWithNames[] = {
+    "ALLIES",
+    "ENEMIES",
+    "STRUCTURES",
+    "SHRUBBERY",
+    "PROJECTILES",
+    "WALLS",
+    "SMALL_MISSILES",
+    "BALLISTIC_MISSILES",
+    "CONTROLLED_STRUCTURES",
+    nullptr,
+};
+
+void Parse_All_Vet_Levels_ASCII_String(INI *ini, void *formal, void *store, const void *user_data)
+{
+    auto OCL_str = ini->Get_Next_Ascii_String();
```

Comment:
Perhaps can curb the use of Auto in these new INI functions.

Reply:
fixed.